### DISCUSSION TITLE
integrate other cars

### DISCUSSION THREAD

**Discussion by eranyanay** [teslamate-org/teslamate#1797] [Q&A]
Hi,
I really like this project and statistics, and I wonder if, given that I already able to pull data from my other car in my own custom way (percentage, driven range etc), if there is an acceptable way to perhaps attach it to teslamate as well? 
As another car?

**Comment by DrMichael** [comment]
Sorry, I don't get it...

You already have teslamate running with one car and want to add another car? Should just work, if it is in your tesla account.
You are pulling data already and want to run teslamate in parallel? Yes, should work. Teslamate does not pull, but subscribes to tesla data.

**Reply by eranyanay** [reply]
Sorry, my initial question is a little bit unclear - I meant other car not being a tesla.
I do have all the stats for it to infer from it the same kind of statistics and data that is collected from tesla API

**Reply by DrMichael** [reply]
Well, you would need to fill the database correctly. Not easily done, I'd say.

**Reply by eranyanay** [reply]
If I manage to update the db accordingly, would that be enough?
How stable the db structure is over time, based on updates etc?

Im willing to think about this option if its quite stable nowdays

**Reply by DrMichael** [reply]
Probably @adriankumpf can tell...
